### DISCUSSION TITLE
What is the default mode?

### DISCUSSION THREAD

**Discussion by 0-harshit-0** [seleniumbase/SeleniumBase#3336] [Q&A] [answered]
if I do: self.open(url) will this automatically go to self.cdp.open(url) mode?

**Comment by mdmintz** [comment, marked as answer]
The default mode is no stealth.

If [CDP Mode](https://github.com/seleniumbase/SeleniumBase/blob/master/examples/cdp_mode/ReadMe.md) is already active, then methods such as `self.open(url)` will redirect to the CDP Mode equivalent, such as `self.cdp.open(url)` in that case.
